submissions/bolarinwa_fasanmi/HTML/landingpage.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>landing e-commerce page</title>
+    <link rel="stylesheet" href="style1.css">
+    
+</head>
+
+
+<body>
+    <div class="container">
+        <div class="head">
+            <img src="image/logo.png" alt="logo" id="logo">
+            <input type="text" placeholder="search" id="search">
+            <nav>
+                <ul>
+
+                    <li><a href="#">Cart</a></li>
+
+
+                    <li><a href="#">Sign Up</a></li>
+
+
+                    <li><a href="#">Log In</a></li>
+
+                </ul>
+
+            </nav>
+            </span>
+        </div>
+
+
+        <main>
+
+            <section>
+                <div class="section">
+                    <div id="list">
+                        <ul>
+
+                            <li> <a href="#">Shoes</a> </li>
+                            <li> <a href="#">Bags</a> </li>
+                            <li> <a href="#">Wrist Watches</a> </li>
+                            <li> <a href="#">Smart Watches</a> </li>
+                            <li> <a href="#">Face Caps</a> </li>
+                            <li> <a href="#">Necklace</a></li>
+                            <li> <a href="#">Phones</a></li>
+                            <li> <a href="#">Computers</a></li>
+                            <li> <a href="#">Electronics</a></li>
+                            <li> <a href="#">Home Kits</a></li>
+                            <li> <a href="#">Fashion</a></li>
+                        </ul>
+
+
+                    </div>
+                    <img src="image/flower1.jpg" alt="flower" class="picture">
+
+
+                </div>
+                <div class="box">

```

Comment:
landing page correction on code line 63, close div tag

```suggestion
                <div class="box">
```suggestion
                <div class="box">


``
```
```